src/dirac_cwl_proto/execution_hooks/DataManagement/Sandbox.py

```diff
@@ -0,0 +1,71 @@
+import logging
+import random
+import tarfile
+from pathlib import Path
+from typing import Optional, Sequence
+
+logger = logging.getLogger(__name__)
+
+
+class SandboxStoreClient:
+    """Class used for sandbox operation"""
+
+    def uploadFilesAsSandbox(
```

Comment:
In python, methods/functions should be lower case with `_` (don't look at what we are doing in DIRAC 😅 ):

```suggestion
    def upload_files_as_sandbox(
```